src/budget/budget.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2019 The DeVault developers
+// Copyright (c) 2019 Jon Spock
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#include "budget/budget.h"
+#include "amount.h"
+#include "chainparams.h"
+#include "config.h"
+#include "dstencode.h"
+#include "logging.h"
+#include "primitives/transaction.h"
+#include <consensus/validation.h>
+
+// Group Address with % for safety/clarity
+struct BudgetStruct {
```

Comment:
Wouldn't be good to have another label field to identify the DAO

Reply:
Makes sense